### DISCUSSION TITLE
Panel position

### DISCUSSION THREAD

**Discussion by Robertarduino** [ClassicDIY/SkyeTracker#37] [Q&A] [answered]
Please confirm: panels position in parallel to ground level it's 90dgr in app or 0dgr. For initialization panels go max to east and parallel to ground level. It's correct for my observation?

**Comment by Robertarduino** [comment]
After today observation. If system configured according movie share at YouTube. Park position system moved to east and max elevation in my configuration 54dgr. But during tracking mode system is follow panels in the same elevation dgr as sun. So system is not in 90dgr to sun light. If I reverse polarization and change min and max dgr elevation system works correctly. But in case of anonometer alarm it will open panels max and it will be problem during wind . How to resolve this issue?

**Comment by graham22** [comment, marked as answer]
The panel should face east with the elevation at 0 deg facing the horizon
at the start of the initialization sequence

**Comment by graham22** [comment]
The max/min degrees for the elevation is the angle of the panels when the
actuator is fully extended/retracted ( usually 0 & 90deg) this should not
be the sun's max elevation at your latitude (54deg)

**Reply by Robertarduino** [reply]
In ma construction : 0inch = 4dgr. Max extend =elevation 54dgr.

**Reply by Robertarduino** [reply]
So it's bug in program. After checking, all position confirmed. Sun angle = panel angle. For example sun =58dgr. Panel angle 52dgr. But to get 90dgr sun exposure of  panels it should go to 32dgr. At end of day sun is going down so for exp. 4dgr and panel get also 4dgr. But it should go to max elevation so 54dgr in my construction. If direction of power reserved function seams Ok by wind alarm position will be in opposite. One solution is to change elevation dgr in opposite. But Min. Maximum is 44dgr.  Could you open in app min and max from 0 to90dgr?

**Comment by graham22** [comment]
to clarify, on my tracker
when I say the tracker is at 0° elevation, the panels are facing the
horizon and are vertical and the vertical actuator is fully extended.
When the tracker is at 90° the panels are facing up and are horizontal, the
vertical actuator is fully retracted.
When the anemometer detects wind speeds of ~30 km/hr, it will fully retract
the vertical actuator to minimize the wind pressure with the panels facing
up at 90°.
If on your frame, when the vertical actuator is fully extended and the
tracker is not at 0° but at lets say 15° and when the actuator is fully
retracted and the tracker is not at 90° but lets say 60° then you would
have to enter 15 & 60 in the app so the code can properly calculate the run
time to move the tracker by 1°, the tracker will run the actuators in ~5°
increments as the sun moves

On Sun, Jul 10, 2022 at 7:52 AM Robertarduino ***@***.***>
wrote:

> So it's bug in program. After checking, all position confirmed. Sun angle
> = panel angle. For example sun =58dgr. Panel angle 52dgr. But to get 90dgr
> sun exposure of panels it should go to 32dgr. At end of day sun is going
> down so for exp. 4dgr and panel get also 4dgr. But it should go to max
> elevation so 54dgr in my construction. If direction of power reserved
> function seams Ok by wind alarm position will be in opposite. One solution
> is to change elevation dgr in opposite. But Min. Maximum is 44dgr. Could
> you open in app min and max from 0 to90dgr?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/37#discussioncomment-3115416>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATK2I3C3EAQ4E25WZJDVTK2QJANCNFSM53DQFO5Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331

**Reply by Robertarduino** [reply]
Now it's cleare ;). I measure in opposite. So my system will reach 36dgr fully extend/ vertical position and 86dgr for horizontal. I will continue testing ;) thanks.

**Reply by Robertarduino** [reply]
I think EU vs US drawing behavior appears ;)

**Comment by Robertarduino** [comment]
For better understanding. Final description:
0 dgr position in app - panels in vertical. In real position its in 90dgr to ground level.
90 dgr position  in app - panels in horizontal position. In real flat to ground level, it's 0dgr position. 
Initialization position -  max East and 0dgr position.
Park position - south position. And 84 dgr ( flat position) to prevent from wind. 
Tracking position - according data from sun calculation.
Example: if system going almost to flat position for example 4dgr in app it will be 86dgr MAX elevation. For MIN position if system can reach 54dgr from ground level in app it will be 36dgr for MIN elevation.

**Reply by graham22** [reply]
correct, it's always elevation, 0° is at the horizon
